src/pj_factors.c

```diff
@@ -72,7 +72,7 @@ int pj_factors(LP lp, PJ *P, double h, struct FACTORS *fac) {
 
         /* convergence */
         if (!(fac->code & IS_ANAL_CONV)) {
-            fac->conv = - atan2(fac->der.y_l, fac->der.x_l);
+            fac->conv = - atan2(fac->der.x_p, fac->der.y_p);
```

Comment:
Thanks. Both for taking time to explain and for letting us use your text in the docs.